src/undo/ArrangeUndoAction.cpp

```diff
@@ -0,0 +1,44 @@
+#include "ArrangeUndoAction.h"
+
+#include "model/Element.h"
+#include "model/PageRef.h"
+
+#include "Range.h"
+#include "i18n.h"
+
+ArrangeUndoAction::ArrangeUndoAction(const PageRef& page, Layer* layer, InsertOrder oldOrder, InsertOrder newOrder):
+        UndoAction("ScaleUndoAction"), layer(layer), oldOrder(oldOrder), newOrder(newOrder) {
```

Comment:
```suggestion
        UndoAction("ArrangeUndoAction"), layer(layer), oldOrder(oldOrder), newOrder(newOrder) {
```

Reply:
Thanks for the catch (you can tell I started on these changes by copying `ScaleUndoAction.cpp` :P).